user-guide/live-traffic/live-traffic.md

```diff
@@ -218,15 +239,15 @@ Use the __Replay__ context menu option to replay previously captured sessions.
 
 ### Select
 
-Use the __Select__ context menu option to select sessions that are directly related to the currently selected session. The options are to select __parent__, __children__, and __duplicate__ sessions.
+Use the __Select__ context menu option to select sessions that are directly related to the currently selected session.  The possibilities are to choose  __parent__, __children__, and __duplicate__ sessions.
 
 - __Parent request__ Selects the session that is a parent of the currently selected session. Keyboard shortcut: __P__
-- __Children requests__ Selects the sessions that are children of the currently selected session. Keyboard shortcut: __C__
+- __Children requests__ Select the sessions that are children of the currently selected session. Keyboard shortcut: __C__
```

Comment:
I'm not sure this change is correct - we are talking about context menu item here, so probably it should be Selects. Also it is inconsistent with the rest of the items here.

Reply:
Its a bit confusing but the change is actually grammatically correct it is **Selects the session** (singular) VS **Select the sessions** (plural). This option (_Children requests_) will select multiple sessions and the other are options for singular selection.